### DISCUSSION TITLE
Lazygit show one tab at a time.

### DISCUSSION THREAD

**Discussion by PicInfiniti** [jesseduffield/lazygit#3786] [Q&A]
Hi. My Lazygit show one tab at the time when i use it inside Lazygit. It worked fine until today. But when I open Lazygit in terminal there is no problem.

Inside Lazygit
![Screenshot from 2024-07-30 11-59-38](https://github.com/user-attachments/assets/9b1304a7-6900-47d7-a91c-a415e8d159fc)

inside Terminal
![Screenshot from 2024-07-30 11-59-22](https://github.com/user-attachments/assets/cf775dc2-6e73-4c30-bdb1-f056c27a56ad)

**Comment by mark2185** [comment]
Whichever plugin you're using to launch `lazygit` (I guess you're launching it from `neovim`?) is launching `lazygit` with an argument, check the docs from the plugin.

You can get that zoomed view when you press `+` or `_`.

**Reply by PicInfiniti** [reply]
Thanks for the answer. I used the wrong combination of keys. I used `space - g - l`, but I should have used `space - g - g`.
